x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/analysis/Verifier.java

```diff
@@ -82,11 +82,9 @@ else if (p.resolved()) {
             // handle aggregate first to disambiguate between missing fields or incorrect function declaration
             if (p instanceof Aggregate aggregate) {
                 for (NamedExpression agg : aggregate.aggregates()) {
-                    if (agg instanceof Alias as) {
-                        var child = as.child();
-                        if (child instanceof UnresolvedAttribute u) {
-                            failures.add(fail(child, "invalid stats declaration; [{}] is not an aggregate function", child.sourceText()));
-                        }
+                    var child = Alias.unwrap(agg);
+                    if (child instanceof UnresolvedAttribute u) {
+                        failures.add(fail(child, "invalid stats declaration; [{}] is not an aggregate function", child.sourceText()));
```

Comment:
nit: `u` isn't used. But wondering if it couldn't be put to use:
```suggestion
                    if (Alias.unwrap(agg) instanceof UnresolvedAttribute u) {
                        failures.add(fail(u, "invalid stats declaration; [{}] is not an aggregate function", u.sourceText()));
```